src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -104,6 +104,12 @@ func importConfig(
 				}
 			}
 
+			// if the value is [secret] then it means that we didn't change the
+			// secret field value, and it's fine to leave it like that and avoid
+			// printing that we change it from [secret] to [redacted].
```

Comment:
This comment partially duplicates the comment on line L117 and the first half of it refers to keeping the sentinel value, where the conditional is about changing the value if it isn't a sentinel. wdyt about something like,

```
// If the value is not [secret], the user changed the redacted sentinel
// value and is changing the secret. We redact the value that we store
// to our to-be-printed propertyDeltas.
```

Also, this is redacting the value a bit too early -- the `v` here is saved into `upsert`, meaning the value will be saved as the value `[redacted]`. the modification needs to happen to what is added to `propertyDeltas`, probably in the first two places below.

Reply:
Also the TODO below on L120 can probably be deleted -- perhaps there's something about the typing that is different within the `interface{}`, but that's ok.